.nyc_output/processinfo/index.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
Great work @kechyy, could you please remove the .nyc_output folder 
```git rm -rf .nyc_output``` would work